java/springboot_hikaricp_using_dsql_connector/src/main/java/org/example/Example.java

```diff
@@ -0,0 +1,241 @@
+package org.example;
```

Comment:
This `Example.java` file seems to be exactly the same as the `pgjdbc_hikaricp_using_dsql_connector` `Example.java` file.

I'm not sure the intention, but I think maybe it's not needed at all given the `SpringBootHikariExampleApplication.java` file?

That being said, I'm not sure how we will exercise that file in the GitHub workflow given there is no test entry point and this `Example.java` file doesn't seem to reference the Spring Boot application at all.